<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>6</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>설정</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_22">
          <item row="0" column="0">
           <widget class="QToolBox" name="toolBox">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>758</width>
               <height>205</height>
              </rect>
             </property>
             <attribute name="label">
              <string>ON</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>온도/습도</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <layout class="QGridLayout" name="gridLayout_3">
                    <item row="1" column="1">
                     <widget class="QDoubleSpinBox" name="h_on_set">
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                      <property name="decimals">
                       <number>1</number>
                      </property>
                      <property name="value">
                       <double>80.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QDoubleSpinBox" name="t_on_set">
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                      <property name="decimals">
                       <number>1</number>
                      </property>
                      <property name="value">
                       <double>25.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>습도</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string>온도</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="on_set">
                    <property name="text">
                     <string>완료 </string>
                    </property>
                    <property name="autoDefault">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>758</width>
               <height>205</height>
              </rect>
             </property>
             <attribute name="label">
              <string>OFF</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_24">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_17">
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_20">
                  <item row="0" column="0">
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="off_set">
                    <property name="text">
                     <string>완료 </string>
                    </property>
                    <property name="autoDefault">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_18">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_17">
                    <property name="text">
                     <string>온도/습도</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <layout class="QGridLayout" name="gridLayout_19">
                    <item row="1" column="1">
                     <widget class="QDoubleSpinBox" name="h_off_set">
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                      <property name="decimals">
                       <number>1</number>
                      </property>
                      <property name="value">
                       <double>50.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QDoubleSpinBox" name="t_off_set">
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                      <property name="decimals">
                       <number>1</number>
                      </property>
                      <property name="value">
                       <double>20.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_18">
                      <property name="text">
                       <string>습도</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_19">
                      <property name="text">
                       <string>온도</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>현재상태</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>미세먼지</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>온도</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QLabel" name="t_label">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QLineEdit" name="time">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>습도</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="d_image">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QLabel" name="h_label">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>유해가스</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="d_label">
              <property name="font">
               <font>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="g_label">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="g_image">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="exit">
      <property name="text">
       <string>종료</string>
      </property>
      <property name="iconSize">
       <size>
        <width>6</width>
        <height>24</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>on_set</sender>
   <signal>clicked()</signal>
   <receiver>h_on_set</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>on_set</sender>
   <signal>clicked()</signal>
   <receiver>t_on_set</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>off_set</sender>
   <signal>clicked()</signal>
   <receiver>t_off_set</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>off_set</sender>
   <signal>clicked()</signal>
   <receiver>h_off_set</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
